### ISSUE TITLE
Support to add the python module path interface to jython extension

### ISSUE THREAD

**Issue by jx6f** [zaproxy/zaproxy#2980]
I'm developing a ZAP script using jython.
We can use python modules installed by the package manager such as pip.
Here is the example code.
```python
import sys
sys.path.append('/path/to/modules/')
import some_module_installed_by_pip
```

In order to improve the portability of scripts, I want to propose to support the interface to add the python module path.

I confirmed that the following codes works fine.
```
diff --git a/src/org/zaproxy/zap/extension/jython/ExtensionJython.java b/src/org/zaproxy/zap/extension/jython/ExtensionJython.java
index 5d76920..fef192b 100644
--- a/src/org/zaproxy/zap/extension/jython/ExtensionJython.java
+++ b/src/org/zaproxy/zap/extension/jython/ExtensionJython.java
@@ -56,6 +56,7 @@ public class ExtensionJython extends ExtensionAdaptor {
                ScriptEngine se = mgr.getEngineByExtension("py");
 
                if (se == null) {
+                       System.setProperty("python.path","/usr/lib/python2.7/site-packages/");
                        PyScriptEngineFactory factory = new PyScriptEngineFactory();
                        this.getExtScript().registerScriptEngineWrapper(new JythonEngineWrapper(factory.getScriptEngine()));
                }
```

After I implement the configuration screen, I would like to send a PR.
I'm going to implement the OptionPanel for jython extension, but there is only one configuration.
Is it better to integrate with other extension's OptionPanel such as script extension?
Could you give me some advice?

**Comment by thc202** [comment]
Having just one option is not a problem (e.g. JVM, Search...), using a new options panel is probably better anyway (option(s) related to the add-on are kept in the same place).

**Comment by psiinon** [comment]
This would be a really good enhancement - we should look at doing similar things for the other scripting languages we support (if relevant)

**Comment by jx6f** [comment]
Thank you for your replies.
@thc202 I'll implement a new option panel.
@psiinon I also reseached about jruby, but I couldn't find.

**Comment by thc202** [comment]
Fixed in zaproxy/zap-extensions#614.

**Comment by thc202** [comment]
Released in version 5 of Python scripting add-on.
